e2e-tests/src/e2eTest/kotlin/com/r3/corda/ledger/utxo/e2etest/IdentifiableTests.kt

```diff
@@ -238,8 +235,8 @@ class IdentifiableTests {
         assertThat(response.flowStatus).isEqualTo(REST_FLOW_STATUS_FAILED)
         assertThat(response.flowError?.message)
             .contains(
-                "On identifiable state(s) updating, each created identifiable state's identifier must match one consumed identifiable " +
-                        "state's state ref or identifier, exclusively."
+                "On identifiable state(s) updating, only one identifiable state with a matching identifier must be consumed for every " +
```

Comment:
The messaging for this rule was never particularly clear to understand, but it should convey that:

 - A consumed identifier must exist exactly once as an input
   - i.e. you cannot consume the same state twice
   - no two states can have the same identifier
 - A produced identifier must exist exactly once as an output
   - the state cannot be split into two states with the same identifier
 - A produced identifier must exist exactly once as a consumed identifier
   - the previous version of an evolving identifiable state must be consumed

Understandably, not easy to fit all of that detail into a single sentence.

Reply:
It's a bit hard to comprehend but I wouldn't say it's hard to understand. I don't see a better way to phrase this if we want to keep this as a single sentence and not breaking it down into separate bullet points.